static/src/javascripts/bootstraps/commercial.js

```diff
@@ -46,12 +50,36 @@ define([
         ['cm-sliceAdverts', sliceAdverts.init],
         ['cm-frontCommercialComponents', frontCommercialComponents.init],
         ['cm-closeDisabledSlots', closeDisabledSlots.init]
+    ],
+    secondaryModules = [
```

Comment:
I've been told we are moving away from single-var declaration, but I don't feel strongly either way :)